agents/xy_stage/xy_latrt_agent.py

```diff
@@ -0,0 +1,253 @@
+import os
+import argparse
+import time
+import txaio
+
+
+ON_RTD = os.environ.get('READTHEDOCS') == 'True'
+if not ON_RTD:
+    from ocs import ocs_agent, site_config
+    from ocs.ocs_twisted import TimeoutLock, Pacemaker
+
+    ## yes I shouldn't have named that module agent
+    from xy_agent.xy_connect import XY_Stage
+
+class LATRt_XY_Agent:
```

Comment:
Sorry for not noticing this in the first review, but class name should be formatting in CapWords, i.e. `LatrtXyAgent`, without the underscores. This will keep the naming consistent with other Agent class names.